components/pedigree/resources/src/main/resources/pedigree/controller.js

```diff
@@ -261,6 +261,35 @@ define([
                 }
             }
 
+            var callPatientCreate = function(studyName) {
+                var createPatientURL = editor.getExternalEndpoint().getFamilyNewPatientURL();
+                if (studyName && studyName.length > 0) {
+                    createPatientURL += "&studyName=" + studyName;
+                }
+                document.fire("pedigree:blockinteraction:start", {"message": "Waiting for the patient record to be created..."});
+                new Ajax.Request(createPatientURL, {
+                    method: "GET",
+                    onSuccess: function(response) {
+                        if (response.responseJSON && response.responseJSON.hasOwnProperty("newID")) {
+                            console.log("Created new patient: " + Helpers.stringifyObject(response.responseJSON));
+                            onCreated(response.responseJSON.newID)
+                        } else {
+                            onFailureHandler();
+                        }
+                    },
+                    onFailure: onFailureHandler,
+                    onComplete: function() { document.fire("pedigree:blockinteraction:finish"); }
+                });
+            };
+
+            var studies = editor.getPreferencesManager().getConfigurationOption("studies");
+            if (studies.length > 0) {
```

Comment:
I'm not sure, but in case there are required fields to me it seems more logical to first ask for a study, and then ask for required fields (which matches the workflow in PT itself, when a new patient is created first a study selection dialog pops up, then the patient form appears (which would pop up the dialog to enter required fields in the CLIn instance).

Reply:
In the case of pedigree there are input fields presented and here it will be more logical to ask to fill them first, then ask for study. When a new patient is created there is no patient form opened yet. I think current workflow is reasonable.